e2e_tests/data/yolo/to/darknet.labels

```diff
@@ -0,0 +1,2 @@
+bbox
```

Comment:
does yolo really only have a list of classes as output? if so can we just slim down the example json? It would seem not to need anything complex.